articles/sendgrid-dotnet-how-to-send-email.md

```diff
@@ -149,7 +149,83 @@ The following examples show how to send a message using the Web API.
+
+The contents of `appsettings.json` file should look like below  -
+
+    {
+       "Logging": {
+       "IncludeScopes": false,
+       "LogLevel": {
+       "Default": "Debug",
+       "System": "Information",
+       "Microsoft": "Information"
+         }
+       },
+     "SENDGRID_API_KEY": "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+
+As first step we need to add the below code in Startup.cs file of the .NET Core api project. This is required so that we can access the `SENDGRID_API_KEY` from the `appsettings.json` file by using dependency injection in the api controller. The `IConfiguration` interface can be injected at the constructor of controller after adding it in the `ConfigureServices` method below. The content of `Startup.cs` file looks like below after adding the required code -
+
+        public IConfigurationRoot Configuration { get; }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            // Add mvc here
+            services.AddMvc();
+            services.AddSingleton<IConfiguration>(Configuration);
+        }
+
+At the controller after injecting the `IConfiguration` interface we can use the `CreateSingleEmailToMultipleRecipients` method of the `MailHelper` class to send a single email to multiple recipients . It accepts one addtional boolean parameter named `showAllRecipients` . This parameter can be used to control whether email receipients will be able to see each others mail id who are in the `To` section of email. The sample code for controller should be like below 
+
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Mvc;
+    using SendGrid;
+    using SendGrid.Helpers.Mail;
+    using Microsoft.Extensions.Configuration;
+
+    namespace SendgridMailApp.Controllers
+    {
+        [Route("api/[controller]")]
+        public class NotificationController : Controller
+        {
+           private readonly IConfiguration _configuration;
+
+           public NotificationController(IConfiguration configuration)
+           {
+             _configuration = configuration;
+           }      
+        
+           [Route("SendNotification")]
+           public async Task PostMessage()
+           {
+              var apiKey = _configuration.GetSection("SENDGRID_API_KEY").Value;
+              var client = new SendGridClient(apiKey);
+              var from = new EmailAddress("noreply@example.com", "Sendgrid user");
+              List<EmailAddress> tos = new List<EmailAddress>
+              {
+                  new EmailAddress("Example1@testmail.com", "Recipient 1"),
+                  new EmailAddress("Example2@testmail.com", "Recipient 2"),
+                  new EmailAddress("Example3@testmail.com","Recipient 3")
+              };
+            
+              var subject = "Hello world email from Sendgrid ";
+              var htmlContent = "<strong>Hello world in HTML content</strong>";
```

Comment:
Hello world in HTML content > Hello world with HTML content

Reply:
ok i will modify it . Thanks for the input :)